placetobe-invest-page.html

```diff
@@ -403,6 +433,16 @@
 
       _handleInvestmentSignError: function() {
         alert('Sorry, er is iets iets misgegaan bij het valideren van je betaling.')
+      },
+
+      _computeSelectedIncentiveDescription: function (selectedIncentiveId) {
+        var availableIncentives = this.availableIncentives;
```

Comment:
Waarom deze niet als observable meenemen in de computed method? Dan zijn we zeker dat deze op het juiste moment gerunt wordt